### ISSUE TITLE
connection problem that only applies to PCs

### ISSUE THREAD

**Issue by hakova** [jitsi/docker-jitsi-meet#1036]
Hi all,

I made the big mistake of upgrading my working jitsi-meet on docker which broke it again. This has been a consistent experience for me so far, that is why I refer to it as a big mistake. The current version seems to accept connections only from an android or iOS client but a connection cannot be established from PCs that operate on linux or windows via firefox or chrome/chromium. When connection is attempted from PCs it disconnects immediately and every following re-connection attempt fails. 

I have not changed any settings in .env or docker-compose files of the previously working set up, nor have I changed the ports that I forwarded (namely 80/tcp, 443/tcp and 10000/udp). Yes, the PUBLIC_URL and DOCKER_HOST_ADDRESS are defined correctly. LetsEncrypt certificates are downloaded and applied during the first run following the upgrade. I do not have a reverse proxy set up, and the ports are directly forwarded from my edge router to the server.

I am not sure if this has anything to do with my problem, but firefox only requests access to the microphone and not to the webcam, which doesn't turn on during connection attempts. It appears that the request to use webcam has never been received. I tried it with chromium too both in linux and windows with the same failure. 

This is a snippet of the server log with the room name and server address modified:

`jvb_1      | May 02, 2021 12:17:57 PM org.jitsi.utils.logging2.LoggerImpl log
jvb_1      | INFO: Performed a successful health check in PT0S. Sticky failure: false
jvb_1      | May 02, 2021 12:18:07 PM org.jitsi.utils.logging2.LoggerImpl log
jvb_1      | INFO: Performed a successful health check in PT0S. Sticky failure: false
web_1      | 45.76.30.98 - - [02/May/2021:12:18:15 -0500] "GET /room1 HTTP/2.0" 200 21565 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:88.0) Gecko/20100101 Firefox/88.0"
web_1      | 45.76.30.98 - - [02/May/2021:12:18:16 -0500] "GET /css/all.css?v=4900 HTTP/2.0" 200 32424 "https://my.jitsi.server.com/room1" "Mozilla/5.0 (X11; Linux x86_64; rv:88.0) Gecko/20100101 Firefox/88.0"
web_1      | 45.76.30.98 - - [02/May/2021:12:18:16 -0500] "GET /libs/do_external_connect.min.js?v=1 HTTP/2.0" 200 1554 "https://my.jitsi.server.com/room1" "Mozilla/5.0 (X11; Linux x86_64; rv:88.0) Gecko/20100101 Firefox/88.0"
web_1      | 45.76.30.98 - - [02/May/2021:12:18:16 -0500] "GET /libs/lib-jitsi-meet.min.js?v=4900 HTTP/2.0" 200 236648 "https://my.jitsi.server.com/room1" "Mozilla/5.0 (X11; Linux x86_64; rv:88.0) Gecko/20100101 Firefox/88.0"
web_1      | 45.76.30.98 - - [02/May/2021:12:18:16 -0500] "GET /libs/app.bundle.min.js?v=4900 HTTP/2.0" 200 1269426 "https://my.jitsi.server.com/room1" "Mozilla/5.0 (X11; Linux x86_64; rv:88.0) Gecko/20100101 Firefox/88.0"
web_1      | 45.76.30.98 - - [02/May/2021:12:18:16 -0500] "GET /pwa-worker.js HTTP/2.0" 200 1499 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:88.0) Gecko/20100101 Firefox/88.0"
jvb_1      | May 02, 2021 12:18:17 PM org.jitsi.utils.logging2.LoggerImpl log
jvb_1      | INFO: Performed a successful health check in PT0S. Sticky failure: false
web_1      | 45.76.30.98 - - [02/May/2021:12:18:18 -0500] "GET /images/avatar.png HTTP/2.0" 200 5995 "https://my.jitsi.server.com/room1" "Mozilla/5.0 (X11; Linux x86_64; rv:88.0) Gecko/20100101 Firefox/88.0"
web_1      | 45.76.30.98 - - [02/May/2021:12:18:18 -0500] "GET /sounds/e2eeOff.mp3 HTTP/2.0" 206 10389 "https://my.jitsi.server.com/room1" "Mozilla/5.0 (X11; Linux x86_64; rv:88.0) Gecko/20100101 Firefox/88.0"
web_1      | 45.76.30.98 - - [02/May/2021:12:18:18 -0500] "GET /sounds/e2eeOn.mp3 HTTP/2.0" 206 10076 "https://my.jitsi.server.com/room1" "Mozilla/5.0 (X11; Linux x86_64; rv:88.0) Gecko/20100101 Firefox/88.0"
web_1      | 45.76.30.98 - - [02/May/2021:12:18:18 -0500] "GET /sounds/joined.wav HTTP/2.0" 206 42156 "https://my.jitsi.server.com/room1" "Mozilla/5.0 (X11; Linux x86_64; rv:88.0) Gecko/20100101 Firefox/88.0"
web_1      | 45.76.30.98 - - [02/May/2021:12:18:18 -0500] "GET /sounds/left.wav HTTP/2.0" 206 42156 "https://my.jitsi.server.com/room1" "Mozilla/5.0 (X11; Linux x86_64; rv:88.0) Gecko/20100101 Firefox/88.0"
web_1      | 45.76.30.98 - - [02/May/2021:12:18:18 -0500] "GET /xmpp-websocket?room=room1 HTTP/1.1" 403 284 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:88.0) Gecko/20100101 Firefox/88.0"
web_1      | 45.76.30.98 - - [02/May/2021:12:18:18 -0500] "GET /pwa-worker.js HTTP/2.0" 200 1499 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:88.0) Gecko/20100101 Firefox/88.0"
jvb_1      | May 02, 2021 12:18:27 PM org.jitsi.utils.logging2.LoggerImpl log
jvb_1      | INFO: Performed a successful health check in PT0S. Sticky failure: false`

Any help will be greatly appreciated.

**Comment by saghul** [comment]
What do you see in the browser console logs?

**Comment by hakova** [comment]
Thank you for your quick reply. Here is the browser console output:

`1619977036944	Toolkit.Telemetry	ERROR	TelemetrySend::sendPersistedPings - failed to send ping d32ae78d-612e-4e8a-97d2-2223516ec78a: {"isTrusted":true} Log.jsm:723
Unchecked lastError value: Error: Could not establish connection. Receiving end does not exist. 2 background.js:2176
Unchecked lastError value: Error: Could not establish connection. Receiving end does not exist. 2 background.js:2176
Unchecked lastError value: Error: Could not establish connection. Receiving end does not exist. 2 background.js:2176
Unchecked lastError value: Error: Could not establish connection. Receiving end does not exist. 2 background.js:2176
Unchecked lastError value: Error: Could not establish connection. Receiving end does not exist. 2 background.js:2176
Unchecked lastError value: Error: Could not establish connection. Receiving end does not exist. 2 background.js:2176
GETwss://my.jitsi.server.com/xmpp-websocket?room=room1
[HTTP/1.1 403 Forbidden 40ms]

Unchecked lastError value: Error: Could not establish connection. Receiving end does not exist. 2 background.js:2176
Unchecked lastError value: Error: Could not establish connection. Receiving end does not exist. 2 background.js:2176`

**Comment by saghul** [comment]
Do you have a reverse proxy? Did you configure XMPP_CROSS_DONAIN ?

**Comment by hakova** [comment]
No, I do not have a reverse proxy. I did not configure XMPP_CROSS_DOMAIN.

**Comment by saghul** [comment]
I don't know what could be the cause, sorry.
